### PR TITLE
Beta test pull request

### PR DESCRIPTION
**Pull request by narendrakorrapati**
approve

### CODE REVIEW

File: UserManagementV1/src/main/java/com/narendra/bss/config/WebAppInitializer.java [hunk 1]
```diff
@@ -7,7 +7,7 @@
 import org.springframework.web.servlet.support.AbstractAnnotationConfigDispatcherServletInitializer;
 
 import com.narendra.bss.config.security.ForceChangePasswordFilter;
-
+//Test for alpha
```
Comment (korrapatinarendra) [inline UserManagementV1/src/main/java/com/narendra/bss/config/WebAppInitializer.java]: Change it to beta

Comment (narendrakorrapati) [reply UserManagementV1/src/main/java/com/narendra/bss/config/WebAppInitializer.java]: fixed the comment

File: UserManagementV1/src/main/java/com/narendra/bss/config/WebAppInitializer.java [hunk 2]
```diff
@@ -7,7 +7,7 @@
 import org.springframework.web.servlet.support.AbstractAnnotationConfigDispatcherServletInitializer;
 
 import com.narendra.bss.config.security.ForceChangePasswordFilter;
-
+//Test for beta pull request
```
Comment (korrapatinarendra) [inline UserManagementV1/src/main/java/com/narendra/bss/config/WebAppInitializer.java]: remove the pull request

Comment (narendrakorrapati) [reply UserManagementV1/src/main/java/com/narendra/bss/config/WebAppInitializer.java]: fixed it.

File: UserManagementV1/src/main/java/com/narendra/bss/config/WebAppInitializer.java [hunk 3]
```diff
@@ -7,7 +7,7 @@
 import org.springframework.web.servlet.support.AbstractAnnotationConfigDispatcherServletInitializer;
 
 import com.narendra.bss.config.security.ForceChangePasswordFilter;
-
+//Test for beta merging into master
```
Comment (korrapatinarendra) [inline UserManagementV1/src/main/java/com/narendra/bss/config/WebAppInitializer.java:10]: approved

**Review by korrapatinarendra — APPROVED** [review]

**Pull request merged**